xcodeproj/internal/xcode_schemes.bzl

```
@@ -157,7 +177,7 @@ def make_xcode_schemes(bazel_labels):
         A `struct` that can be used as a `bazel_labels` module.
     """
 
-    def _build_action(targets):
+    def _build_action(targets, pre_actions = [], post_actions = []):
```

```suggestion
    def _build_action(targets, *, pre_actions = [], post_actions = []):
```

We should require the labels once the call gets this complex.